src/CodeGeneration.Roslyn.Tests/Helpers/CompilationTestsBase.cs

```diff
@@ -21,7 +21,9 @@ public abstract class CompilationTestsBase
     static CompilationTestsBase()
     {
         // this "core assemblies hack" is from https://stackoverflow.com/a/47196516/4418060
+#pragma warning disable SA1009 // Closing parenthesis should be spaced correctly
```

Comment:
I temporarily disabled the warning to get the build running

Reply:
We should fix it by updating to stylecop.analyzers 1.2.0-beta.164